src/building/how-to-build-and-run.md

```diff
@@ -215,13 +213,13 @@ your local environment:
 
 ```bash
 $ rustc +stage1 -vV
-rustc 1.25.0-dev
+rustc 1.48.0-dev
 binary: rustc
 commit-hash: unknown
 commit-date: unknown
 host: x86_64-unknown-linux-gnu
-release: 1.25.0-dev
-LLVM version: 4.0
+release: 1.48.0-dev
+LLVM version: 11.0
```

Comment:
It's not a big deal since you're updating other things, but there's much point in updating this version - it gets out of date again every 6 weeks and it still gets the point across.

Reply:
I know, but I figure I might as well update it so it's not *so* out-of-date :)